jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt

```diff
@@ -49,7 +49,7 @@ class AwsExplorerRootNode(project: Project) :
     }
 }
 
-abstract class AwsExplorerServiceRootNode(project: Project, value: String, awsIcon: Icon) :
+abstract class AwsExplorerServiceRootNode(project: Project, value: String, awsIcon: Icon?) :
```

Comment:
Was dropping the icon from the service something that UX suggested? I'm not sure this should be optional - we should either have icons on all our service nodes or none of them. Anything else is going to look funny IMO.

Reply:
Ya, that came from branding and Nick. To be honest this was a quick job since Nick was sitting next to me :)

This should not be optional, it should be coded to always pass null to parent